tests/installation/disable_grub_graphics.pm

```diff
@@ -18,11 +18,21 @@ use testapi;
 sub run {
     my ($self) = shift;
 
-    send_key 'alt-c';
-    assert_screen 'inst-overview-options';
+    if (check_var('VIDEOMODE', 'text')) {
+        send_key 'alt-c';
+        assert_screen 'inst-overview-options';
 
-    send_key 'alt-b';
-    assert_screen 'installation-bootloader-config';
+        send_key 'alt-b';
+        assert_screen 'installation-bootloader-config';
+    }
+    else {
+        # Verify Installation Settings overview is displayed as starting point
+        assert_screen "installation-settings-overview-loaded";
+
+        # Select section booting on Installation Settings overview (video mode)
+        send_key_until_needlematch 'booting-section-selected', 'tab';
+        send_key 'ret';
```

Comment:
@rwx788 @OleksandrOrlov  This part comes from  https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f9eb8ef62c0d9a6b743ac890c20348e2ac635313/tests/installation/edit_optional_kernel_cmd_parameters.pm#L26 . I think this could go into some library, but not sure. Could you please recommend yast lib for it?

Reply:
I would leave it as it is, as we rewrite functionality with the REST API, and there we will provide reusable structure. I could not find any suitable place for it, so looks like we can create a new one.

Reply:
@rwx788 Ok, I will leave it as it is now - keeping it for REST API rewrite.

Reply:
Let's fix the needles pipeline and then proceed with merging, it fails not due to your changes, but someone has merged a needle with wrong filename format.